src/components/ReactionContainer/Reaction.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from 'clsx'
+import * as Styled from './Reactions.styled'
+import { reactionMappingObj } from './helpers'
+import { MouseEventHandler } from 'react'
+import { Reaction as ReactionType, ReactionComponentVariant } from './types'
+
+type Props = {
+  reaction: ReactionType
+  isActive?: boolean
+  variant?: ReactionComponentVariant
+  onClick: MouseEventHandler
+}
+
+const Reaction = ({ reaction, variant = 'standard', onClick }: Props) => {
+  const users = reaction.users?.join(', ') || undefined
+
+  return (
+    <Styled.Reaction
+      className={clsx(variant, { active: reaction.isActive })}
+      onClick={onClick}
+      data-tooltip={users}
+    >
+      <div className='inner-content'>
+        {reactionMappingObj[reaction.type]}
+        {reaction.users && <span>{reaction.users.length}</span>}
+      </div>
+    </Styled.Reaction>
```

Comment:
Only the emoji needs moving down 1px
```suggestion
      <div className="inner-content">{reactionMappingObj[reaction.type]}</div>
      {reaction.users && <span>{reaction.users.length}</span>}
```